airbyte-metrics/reporter/src/main/java/io/airbyte/metrics/reporter/ReporterApp.java

```diff
@@ -6,15 +6,70 @@
 
 import io.airbyte.config.Configs;
 import io.airbyte.config.EnvConfigs;
+import io.airbyte.db.Database;
+import io.airbyte.db.instance.configs.ConfigsDatabaseInstance;
 import io.airbyte.metrics.lib.DatadogClientConfiguration;
 import io.airbyte.metrics.lib.DogStatsDMetricSingleton;
 import io.airbyte.metrics.lib.MetricEmittingApps;
+import io.airbyte.metrics.lib.MetricQueries;
+import io.airbyte.metrics.lib.MetricsRegistry;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import lombok.extern.slf4j.Slf4j;
 
+@Slf4j
 public class ReporterApp {
 
-  public static void main(final String[] args) {
+  public static void main(final String[] args) throws IOException, InterruptedException {
     final Configs configs = new EnvConfigs();
+
     DogStatsDMetricSingleton.initialize(MetricEmittingApps.METRICS_REPORTER, new DatadogClientConfiguration(configs));
+
+    final Database configDatabase = new ConfigsDatabaseInstance(
+        configs.getConfigDatabaseUser(),
+        configs.getConfigDatabasePassword(),
+        configs.getConfigDatabaseUrl())
+            .getInitialized();
+
+    final var pollers = Executors.newScheduledThreadPool(4);
+
+    log.info("Starting pollers..");
+    pollers.scheduleAtFixedRate(() -> {
```

Comment:
this is ugly for now. I will properly abstract these in a follow up PR. I didn't want to start too early since I wanted to see how this looks like first.